lib/ZwaveClient.js

```diff
@@ -150,6 +150,10 @@ async function init (cfg, socket) {
   if (cfg.assumeAwake) {
     options.AssumeAwake = cfg.assumeAwake
   }
+  
+  if (cfg.options) {
+    options = {...options, ...cfg.options}
```

Comment:
Maybe using `Object.assign` here is more clear ?

`Object.assign(options, cfg.options)`